src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -14,9 +16,22 @@ interface Props
     Artsy.ContextProps {
   relay?: RelayProp
   artist?: FollowArtistButton_artist
+  tracking?: any
+  trackingData?: any
```

Comment:
can we add a `TrackingData` inferface based on the schema?

Reply:
I left this as any because `FollowArtistButton` can be used outside of tooltips, this allows us to pass necessary tracking data based on context.